agent/utils/venv_ops.py

```
@@ -11,9 +11,9 @@
 # [配置区] 环境与依赖设定
 # =========================================================
 # MAAFW的py库版本需要手动指定,与C++库版号一致。
-DEV_MAAFW_VERSION = "5.9.2" 
+DEV_MAAFW_VERSION = "5.11.1" 
 # 精确安装失败时的回退范围。更新上方版本时，请务必同步更新此处！
```

Ruff (0.16.0)

[warning] Comment contains ambiguous `，` (FULLWIDTH COMMA). Did you mean `,` (COMMA)?

(RUF003)